### DISCUSSION TITLE
Multi Tenancy

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5399] [Help]
How can I use multitenancy such that I want to add tenants in tenants ? I have reviewed previous discussions but could not find complete information.

**Comment by TechAspirant** [comment]
An idea on this ? You found the solution ?

**Reply by edwardch** [reply]
It might be better to upvote the existing question rather than creating a duplicate one. It results in all relevant information being in one place so anyone looking in future can get the full context of the discussion.
